src/components/tabs/tabs.component.js

```diff
@@ -46,41 +44,19 @@ const Tabs = ({
   const [tabsErrors, setTabsErrors] = useState({});
   const [tabsWarnings, setTabsWarnings] = useState({});
   const [tabsInfos, setTabsInfos] = useState({});
-  const _window = Browser.getWindow();
-  const isInSidebar = !!(sidebarContext && sidebarContext.isInSidebar);
+  const isInSidebar = !!sidebarContext?.isInSidebar;
 
   useLayoutEffect(() => {
     if (firstRender.current) {
```

Comment:
This check using `firstRender` could be replaced by an empty dependency array so it only runs on the first render.

Don't have to change it here because it's not part of your changes, but if you want to tidy that up it's up to you

Reply:
yeah good point I'll update it now the dependencies have been reduced

Reply:
The dependencies are irrelevant because we only want this stuff to run on first render, they're only there to trigger re-runs of the function when the dependencies change

You might have to disable an eslint rule to have the empty array but it's the right thing to do here!

Reply:
Yeah sorry I meant they were reduced down from what they were previously so it's worth updating the layoutEffect further rather than leaving it as is for now etc